packages/augur-test/libs/ContractAPI.ts

```diff
@@ -229,6 +249,13 @@ export class ContractAPI {
     await market.doInitialReport(payoutNumerators, "");
   }
 
+  public async getInitialReporterStake(market: GenericAugurInterfaces.Market<ethers.utils.BigNumber>, payoutNumerators: Array<ethers.utils.BigNumber>): Promise<ethers.utils.BigNumber> {
+    const payoutDistributionHash = await this.derivePayoutDistributionHash(market, payoutNumerators);
+    const initialReporterAddress = await market.getCrowdsourcer_(payoutDistributionHash);
+    const initialReporter = this.augur.contracts.getReportingParticipant(initialReporterAddress);
+    return await initialReporter.getStake_();
+  }
+
   public async finalizeMarket(market: GenericAugurInterfaces.Market<ethers.utils.BigNumber>): Promise<void> {
     await market.finalize();
```

Comment:
Unsure if that does anything?

Reply:
This calls the `finalize` function on the Market.sol contract. There's no return value that it needs to return, though.